monitoring-satellite/monitoring-satellite.libsonnet

```
@@ -14,6 +14,7 @@ local gitpod = import '../components/gitpod/gitpod.libsonnet';
 (import '../addons/networkpolicies-disabled.libsonnet') +
 (if std.objectHas(config, 'alerting') then (import '../addons/alerting.libsonnet')(config) else {}) +
 (if std.objectHas(config, 'remoteWrite') then (import '../addons/remote-write.libsonnet')(config) else {}) +
+(if std.objectHas(config, 'prometheus') then (import '../addons/remote-write-ingress.libsonnet')(config) else {}) +
```

```suggestion
(if (std.objectHas(config, 'prometheus') && (std.objectHas(config.prometheus, 'DNS')) then (import '../addons/remote-write-ingress.libsonnet')(config) else {}) +
```

The field `config.prometheus` is used in several places, I think we need an ingress only when Prometheus is accessible via internet 🙂 


This would also solve one of our checks :)